src/robusta/core/model/base_params.py

```diff
@@ -77,6 +77,7 @@ class ResourceInfo(BaseModel):
     kind: Optional[str]
     node: Optional[str]
     container: Optional[str]
+    cluster: Optional[str]
```

Comment:
Since we already have the cluster, I think it's better to get it from the runner
It's less bug prone